packages/matplotlib/patches/disable_macosx_backend.patch

```diff
@@ -1,15 +1,13 @@
-diff -u matplotlib-2.2.3.bak/setupext.py matplotlib-2.2.3/setupext.py
---- a/setupext.py	2019-10-15 10:09:56.000000000 -0400
-+++ b/setupext.py	2019-10-15 11:24:53.000000000 -0400
-@@ -1798,6 +1798,7 @@
+diff --git a/setupext.py b/setupext.py
+--- a/setupext.py
++++ b/setupext.py
+@@ -665,6 +665,7 @@ class BackendMacOSX(OptionalPackage):
      name = 'macosx'
  
-     def check_requirements(self):
-+        raise CheckFailed("MacOSX backend disabled")
+     def check(self):
++        raise Skipped("MacOSX backend disabled")
          if sys.platform != 'darwin':
```

Comment:
Maybe 
```suggestion
         if sys.platform != 'darwin' or "PYODIDE_PACKAGE_ABI" in os.environ:
```
and we could also try to upstream this fix. Is there a need to rename this method?

Reply:
You mean the check method?

Reply:
Yes, why rename it from `check_requirements` to `check`?

Reply:
The rename comes [from upstream](https://github.com/matplotlib/matplotlib/commit/aceddbfa769d23479608be262680c436c4bb4dc3).